Sources/Jaeger/Jaeger/Agent/CoreData/CoreDataSpan+CoreDataClass.swift

```diff
@@ -23,7 +23,10 @@ class CoreDataSpan: NSManagedObject {
      */
     @discardableResult
     static func create(in context: NSManagedObjectContext, startTime: Date, data: Data) -> CoreDataSpan {
-        let entity = NSEntityDescription.entity(forEntityName: "CoreDataSpan", in: context)!
+        guard let entity = NSEntityDescription.entity(forEntityName: "CoreDataSpan", in: context) else {
+            fatalError("")
```

Comment:
Like the `fatalError` here, needs a message string though…

Reply:
indeed! thx.